server/images.go

```diff
@@ -40,18 +40,17 @@ type RegistryOptions struct {
 }
 
 type Model struct {
-	Name          string `json:"name"`
```

Comment:
this json dates back to 
https://github.com/jmorganca/ollama/commit/2fb52261adb031039fe927350a257da84b9a5854

It does not appear to be used anywhere

Reply:
Remove the field if unused?

Reply:
Sorry I meant `json` part was unused, not the field itself